### DISCUSSION TITLE
DBFirst的Generator不支持MSAccess吗？

### DISCUSSION THREAD

**Discussion by wakuflair** [dotnetcore/FreeSql#1210] [General]
用FreeSql.Generator生成一个MSAccess数据库的model类，报以下错误：

`unsupported type: "MSAccess"`

**Comment by 2881099** [comment]
是的，MsAccess 未实现 IDbFirst 接口，其他数据库基本都实现了。

**Comment by 2881099** [comment]
MsAccess 在 CodeFirst 方面花了很大代价，DbFirst 不敢再碰。
